.github/workflows/ci.yml

```diff
@@ -105,6 +105,13 @@ jobs:
           cache: false
           cache-dependency-path: |
             go.sum
+      # necessary for the fips base image
+      - name: Log in to the Elastic Container registry
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
```

Comment:
Unfortunately, this won't work on forked PRs, GH secrets are inaccessible in that case.

The `test-package-and-push:` job is the one that can access the GH secrets. Is it possible to skip the FIPS when running `make package-snapshot`?